docs/restore-individual-tables.md

```diff
@@ -1,76 +1,105 @@
+First, create the table using the `innodb_file_per_table` setting. This setting tells MySQL to create a separate `.ibd` file for each table in your source directory. Run this command to find the created file in your system:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-During the `--prepare` step, add the [--export] option to the
-command. For example:
+Next, when you run the `--prepare` phase, add the [--export] option to your command. This option tells XtraBackup to create any extra files needed for moving the table:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-When restoring an encrypted InnoDB tablespace table, add the keyring file:
+If you're working with encrypted tables (ones that use transparent data encryption), you'll need extra security settings. You must include the keyring file that has the encryption keys:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/tmp/table \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
 ```
 
-The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the `.cfp` file, which contains the transfer key and an encrypted tablespace key.
-
-The files are located in the target directory:
+After these commands finish successfully, you'll need these files for the table import.
 
 ```text
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
+/data/backups/mysql/test/export_test.cfp
 ```
 
 ## Import the table
 
-Import the table by doing the following:
+Now that we've exported the table, we can move it to the destination system. You need to follow these steps in order on the target server:
+
+* Use the saved schema definition to create an empty table at the destination. This provides the framework for your imported data.
+
+* Copy all three important files (.ibd, .cfp, and .cfg) from your source directory to the matching directory in the destination system.
 
-* Create a copy of the table
-* Discard the table's original tablespace
-* Copy the `.ibd` and `.cfg` files to the location
-* Import the tablespace
 
-To import that table, do the following steps:
+The following is the detailed process:
 
-1. On the destination server, running Percona Server for MySQL with XtraDB or MySQL 8.0, create a table with the same structure.
+1. On your target server (which must be running Percona Server for MySQL with XtraDB or MySQL 8.0), use the schema and create an empty copy of the table in the new location.
 
-2. Run the `ALTER TABLE ... DISCARD TABLESPACE` command to discard the new table's tablespace:
+    ```{.bash data-prompt="mysql>"}
+    mysql> mysql -uroot < ./schema_dumps/test/test.sql;
+    mysql> mysql -uroot < ./schema_dumps/test/t1.sql;
```

Comment:
Option 2:

during export, we used name export_test. lets use that.

< schema_dumps/test/export_test.sql